lint/clang_tidy.bzl

```diff
@@ -270,6 +273,12 @@ def _get_compiler_args(ctx, compilation_context, srcs):
         args.append("-D" + define)
     for define in compilation_context.local_defines.to_list():
         args.append("-D" + define)
+    if hasattr(ctx.rule.attr, "defines"):
```

Comment:
hmm, is it a bug that the compilation_context didn't already supply this local_define value? @jsharpe in case you have an opinion

Reply:
Certainly seems like it, but given its the behaviour of bazel releases then we've got to fix up for it; there shouldn't be too much harm (other than line length and the associated increase in memory use) in adding the defines to the commandline multiple times?